examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,805 @@
+    "- The model outputs a single number (here we use sigmoid function to generate probabilities) which can be interpreted as a likelihood of a certain user clicking on an ad, watching a movie, or viewing a news page.\n",
+    "\n",
+    "\n",
+    "Steps:<br>\n",
+    "* Change the model to `DLRMModel`\n",
+    "* Rerun the pipeline from there from model.fit"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "e02eeaae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = mm.DLRMModel(\n",
+    "    schema,\n",
+    "    embedding_dim=64,\n",
+    "    bottom_block=mm.MLPBlock([128, 64]),\n",
+    "    top_block=mm.MLPBlock([128, 64, 32]),\n",
+    "    prediction_tasks=mm.BinaryClassificationTask(target_column, metrics=[tf.keras.metrics.AUC()])\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "6e39bae2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2442/2442 [==============================] - ETA: 0s - auc_2: 0.6560 - loss: 0.1864 - regularization_loss: 0.0000e+00 - total_loss: 0.1864"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-03-16 15:38:43.261571: W tensorflow/core/grappler/optimizers/loop_optimizer.cc:907] Skipping loop optimization for Merge node with control input: cond/then/_0/cond/cond/branch_executed/_158\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2442/2442 [==============================] - 151s 58ms/step - auc_2: 0.6560 - loss: 0.1864 - regularization_loss: 0.0000e+00 - total_loss: 0.1864 - val_auc_2: 0.5646 - val_loss: 0.1284 - val_regularization_loss: 0.0000e+00 - val_total_loss: 0.1284\n",
+      "CPU times: user 5min 11s, sys: 51.8 s, total: 6min 3s\n",
+      "Wall time: 2min 31s\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<keras.callbacks.History at 0x7f0d6e466070>"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "opt = tf.keras.optimizers.Adagrad(learning_rate=LR)\n",
+    "model.compile(optimizer=opt, run_eagerly=False)\n",
+    "model.fit(train_dl, validation_data=test_dl)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "98efae3b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "save_results(\"DLRM\", model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c491573f",
+   "metadata": {},
+   "source": [
+    "### DCN Model"
```

Comment:
Which DCN model do we implement/train? The left or right?

Reply:
I will add. It is stacked by default. Thanks.